package.json

```diff
@@ -38,7 +38,9 @@
   },
   "devDependencies": {
     "@apollo/client": "3.7.5",
+    "@apollo/gateway": "https://pkg.csb.dev/apollographql/federation/commit/166fdd24/@apollo/gateway",
```

Comment:
Would be nice to avoid this but I guess the gateway release process is slow.

Reply:
This will become the actual gateway version once the patch is released, won't land this until then